packages/python-packages/apiview-copilot/src/_models.py

```diff
@@ -111,6 +111,9 @@ class Comment(BaseModel):
         description="the suggested code which fixes the bad code. If code is not feasible, a description is fine."
     )
     comment: str = Field(description="the contents of the comment.")
+    correlation_id: Optional[str] = Field(
```

Comment:
can we add this to our tsp 😄 ? 

it has been pretty useful!

Reply:
I added it in this PR: https://github.com/Azure/azure-sdk-tools/pull/12254